extensions/wikia/PortableInfobox/services/Parser/MediaWikiParserService.php

```diff
@@ -13,9 +13,14 @@ public function __construct( \Parser $parser, \PPFrame $frame ) {
 	}
 
 	public function parse( $wikitext ) {
-		return $this->getParserInstance()
+		if ( substr( $wikitext, 0, 1 ) == "*" ) {
+			//fix for first item list elements
```

Comment:
It seems clear what this does technically, but why this addition of newline when wikitext begins with a * is necessary?